native/ios/WatermelonDB/Database.swift

```diff
@@ -32,6 +34,7 @@ class Database {
     }
 
     func inTransaction(_ executeBlock: () throws -> Void) throws {
+        fmdb.setKey(self.password)
```

Comment:
It's not clear to me why it was necessary to call setKey every time and I worry about performance with this. I think we want to figure this out before merging. Originally I tried calling `setKey` only in the `open` method but it did not encrypt the database - the method gets called, but it is effectively ignored by SQLite.

Reply:
yup, agreed, should be figured out before merge, I don't like this either